physics/smoke_dust/plume_data_mod.F90

```diff
@@ -45,7 +45,7 @@ module plume_data_mod
   integer, parameter :: savannah        = 3
   integer, parameter :: grassland       = 4
   integer, parameter :: nveg_agreg      = 4
-  integer, parameter :: wind_eff        = 1
+!  integer, parameter :: wind_eff        = 1
```

Comment:
Remove commented line.

Reply:
@dustinswales Sure, I will remove this commented line.